src/index.html

```diff
@@ -1,15 +1,171 @@
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>HTML Form</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+          method="post"
+          autocomplete="off">
+      <fieldset >
+        <legend>
+          Personal information
+        </legend>
+        <div>
+          <label for="surname">
+            Surname:
+            <input type="text"
+                   name="surname"
+                   id="surname">
+          </label>
+        </div>
+        <div>
+          <label for="name">
+            Name:
+            <input type="text"
+                   name="name"
+                   maxlength="15"
+                   id="name">
+          </label>
+        </div>
+        <div>
+          <label class="resize" for="age">
+            How old are You?
+            <input type="number"
+                   value="12"
+                   name="age"
+                   min="10"
+                   id="age">
+          </label>
+        </div>
+        <div>
+          <label for="date">
+            Full date of birth:
+            <input type="date"
+                   name="date"
+                   max="2145-01-08"
+                   id="date">
+          </label>
+        </div>
+        <div>
+          <label for="confirm">
+            I accept the term of the agreement
+            <input type="checkbox"
+                   name="confirm"
+                   id="confirm">
+          </label>
+        </div>
+      </fieldset>
+      <fieldset >
+        <legend>
+          Registration
+        </legend>
+        <div>
+          <label for="email">
+            E-mail:
+            <input type="email"
+                   name="email"
+                   required
+                   placeholder="email@example.com"
+                   id="email">
+          </label>
+        </div>
+        <div>
+          <label for="password">
+            Password:
+            <input type="password"
+                   name="password"
+                   minlength="10" id="password">
+          </label>
+        </div>
+      </fieldset>
+      <fieldset>
+        <legend>
+          An interesting fact about you!
+        </legend>
+        <div>
+            Do you love cats?
+          <label for="yes">
+            <input type="radio"
+                     name="cats"
+                     id="yes">
+              Yes
+          </label>
```

Comment:
обычно редактируют так
```suggestion
          <label for="yes">
            <input 
              type="radio"
              name="cats"
              id="yes"
            >
              Yes
          </label>
```